package.json

```diff
@@ -50,6 +50,7 @@
 		"@automattic/wpcom-block-editor": "file:apps/wpcom-block-editor",
 		"browserslist": "4.8.6",
 		"i18n-calypso": "file:packages/i18n-calypso",
+		"loader-utils": "1.2.3",
```

Comment:
Why is this change necessary?

Reply:
I was trying to fix the test but it didn't work. Will revert it.